modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-enable-node-scan-scannerv4_{context}"]
+= Enabling {op-system} node scanning with Scanner V4
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+
+:FeatureName: RHCOS node scanning with Scanner V4
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* For scanning {op-system} node hosts of the secured cluster, you must have installed the following:
+** Secured Cluster services on {ocp} {ocp-supported-version} or later
+** {product-title-short} version 4.6 or later
++
+For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix]. For life cycle support information for {product-title-short}, see the link:https://access.redhat.com/support/policy/updates/rhacs[{product-title} Support Policy].
+
+.Procedure
+
+To enable node indexing, or node node scanning on Scanner V4, you must set the `ROX_NODE_INDEX_ENABLED` variable to `true` on Central and on Compliance.
+
+. On the cluster where Central is installed, run the following commands:
++
+[source,terminal]
+----
+$ kubectl -n stackrox set env deployment/central ROX_NODE_INDEX_ENABLED=true
+----
++
+[source,terminal]
+----
+$ kubectl -n stackrox set env daemonset/collector ROX_NODE_INDEX_ENABLED=true
+----
+. To verify that node scanning is working, examine the Central logs for the following message:
++
+[source,text]
+----
+Scanned index report and found <number> components for node <node_name>.
```

Comment:
@Maddosaurus do we have any kind of logs in Central or Compliance which may say something like "using Scanner V4 for RHCOS node scanning"? That might be better to look for

Reply:
On Compliance we have a startup log that says "Node Index v4 enabled" ([link](https://github.com/stackrox/stackrox/blob/4c6acfddd54b1002dea6e12034776da77447c852/compliance/compliance.go#L103)).
That log line only ensures that the index is being generated. As both, Node Indexing and Node Scanning, are message-based, there is no single log line that will make it obvious things generally work.